src/components/OptionsList/index.native.tsx

```diff
@@ -0,0 +1,18 @@
+import React, {forwardRef} from 'react';
+import type {ForwardedRef} from 'react';
+import {Keyboard} from 'react-native';
+import BaseOptionsList from './BaseOptionsList';
+import type {OptionsListProps, OptionsList as OptionsListType} from './types';
+
+function OptionsList(props: OptionsListProps, ref: ForwardedRef<OptionsListType>) {
+    return (
+        <BaseOptionsList
+            // eslint-disable-next-line react/jsx-props-no-spreading
+            {...props}
+            ref={ref}
+            onScrollBeginDrag={() => Keyboard.dismiss()}
+        />
+    );
+}
+
```

Comment:
It appears that the displayName was unintentionally removed.
```suggestion

OptionsList.displayName = 'OptionsList';

```